### PR TITLE
fixes after compose

### PR DESCRIPTION
**Pull request by Boobl1k**
I guess there is no way to have only 1 copy of proto directory. We cannot go outside of context in dockerfiles, even with volume based mounts (it works only for final layer)
Actually even if we find the way to do that, it is not ok when service uses something except its own files or dependencies
So I am sure we need to have a copy for each service

### CODE REVIEW

File: api/Program.cs [hunk 1]
```diff
@@ -37,11 +37,17 @@
 
 var app = builder.Build();
 
-await using (var scope = app.Services.CreateAsyncScope())
+migrate:
```
[the remark this answers is not in the document]
Comment (Focus1337) [reply api/Program.cs:40]: Use Polly Retry Policy

[the remark this answers is not in the document]
Comment (Focus1337) [reply api/Program.cs:40]: 5 минут на решение, уебище

[the remark this answers is not in the document]
Comment (DmitryTXCMN) [reply api/Program.cs:40]: > Use Polly Retry Policy

Критикуешь - предлагай, уебище

[the remark this answers is not in the document]
Comment (Focus1337) [reply api/Program.cs]: Replaced "goto". [Commit](https://github.com/Boobl1k/chat-grpc/pull/17/commits/fda3280c7a24be76eefa40448c21024e4505b8ef).

Done.

**Review by Focus1337 — APPROVED** [review]

**Pull request merged**